src/sentry/static/sentry/app/components/version.jsx

```diff
@@ -1,20 +1,42 @@
 import React from 'react';
 import {Link} from 'react-router';
 
+import HoverCard from './hoverCard';
+
 const Version = React.createClass({
   propTypes: {
     anchor: React.PropTypes.bool,
     version: React.PropTypes.string.isRequired,
     orgId: React.PropTypes.string.isRequired,
-    projectId: React.PropTypes.string.isRequired
+    projectId: React.PropTypes.string.isRequired,
+    onIssuePage: React.PropTypes.bool,
```

Comment:
Rename `onIssuePage` to `hasHovercard` or something more generic so it could be used elsewhere.

The `Version` component shouldn't know what an "issue page" is – it's just a self-contained component.